src/SFML/Graphics/Shader.cpp

```diff
@@ -456,19 +393,19 @@ unsigned int sfShader_getNativeHandle(const sfShader* shader)
 ////////////////////////////////////////////////////////////
 void sfShader_bind(const sfShader* shader)
 {
-    sf::Shader::bind(shader ? &shader->This : NULL);
+    sf::Shader::bind(shader ? &shader->This : nullptr);
 }
 
 
 ////////////////////////////////////////////////////////////
-sfBool sfShader_isAvailable(void)
+bool sfShader_isAvailable(void)
 {
-    return sf::Shader::isAvailable() ? sfTrue : sfFalse;
+    return sf::Shader::isAvailable();
 }
 
 
 ////////////////////////////////////////////////////////////
-sfBool sfShader_isGeometryAvailable(void)
+bool sfShader_isGeometryAvailable(void)
 {
-    return sf::Shader::isGeometryAvailable() ? sfTrue : sfFalse;
+    return sf::Shader::isGeometryAvailable();
```

Comment:
Does C++ `bool` implicitly convert to C `bool`?

Reply:
Yes it does. As far as I'm aware they're indistinguishable.